app/controllers/support_interface/change_course_controller.rb

```diff
@@ -1,5 +1,7 @@
 module SupportInterface
   class ChangeCourseController < SupportInterfaceController
+    before_action :make_sure_course_can_be_added, only: %w[select_course_to_add add_course]
```

Comment:
Another point of evidence for @stevehook suggestion to split up the controllers https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1880#discussion_r409413573.